<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFEAcquisitiontestDialog</class>
 <widget class="QWidget" name="QFEAcquisitiontestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;b&gt;Acquisition Device:&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFAcquisitionComboBox" name="cmbDevice"/>
       </item>
       <item>
        <widget class="QToolButton" name="btnConnect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Acquisition Configuration:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFAcquisitionConfigEditorWidget" name="widAcquisitionConfig" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;b&gt;Logging:&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QtLogFile" name="log" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>10</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFAcquisitionComboBox</class>
   <extends>QComboBox</extends>
   <header>qfacquisitioncombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFAcquisitionConfigEditorWidget</class>
   <extends>QWidget</extends>
   <header>qfacquisitionconfigcombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtLogFile</class>
   <extends>QWidget</extends>
   <header>qtlogfile.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
